packages/navcookiebanner/src/react/index.tsx

```diff
@@ -51,9 +51,10 @@ const NavCookieBanner = React.forwardRef<HTMLDivElement, NavCookieBannerProps>(
     function dismiss() {
       const oneYear = new Date()
       oneYear.setFullYear(oneYear.getFullYear() + 1)
+      const domain = '.' + document.domain.replace(/^[^.]+\.(.*\..*)$/, '$1')
```

Comment:
Let's add a comment above this line explaining the regex in a single sentence since it's so abstract. 👍